plugins/net_plugin/net_plugin.cpp

```diff
@@ -3723,7 +3718,7 @@ namespace eosio {
          bool exception = false;
          bool best_head = false;
          bool unlinkable = false;
-         sync_manager::closing_mode close_mode = sync_manager::closing_mode::handshake;
+         sync_manager::closing_mode close_mode = sync_manager::closing_mode::immediately;
          try {
             EOS_ASSERT(ptr->timestamp < (fc::time_point::now() + fc::seconds(7)), block_from_the_future,
                        "received a block from the future, ignoring it: ${id}", ("id", id));
```

Comment:
We will reject the block if this EOS_ASSERT happens. Should `ignoring` be changed to `rejecting`?